### PR TITLE
Resolve #15 -- Resolve downloading label issue

### LINKED ISSUE #15: Fix label issue during downloading of client
image
Fix label during downloading of client

### PR DESCRIPTION
**Pull request by Kipero**
Resolve #15

### CODE REVIEW

**Comment by Kipero** [comment]
Fixed:
image

**Review by lzardy — CHANGES_REQUESTED** [review]
Some unused events.

File: League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs [hunk 1]
```diff
@@ -457,5 +457,15 @@ private void BrowseButton_Click(object sender, EventArgs e)
                     installDirectoryText.Text = selectPath.SelectedPath;
             }
         }
+
+        private void cloningProgressLabel_Click(object sender, EventArgs e)
```
Comment (danil179) [inline League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs]: Delete this and its corresponding event. If you think that these are important please add `TODO:` that explains what is planned here.

File: League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs [hunk 2]
```diff
@@ -457,5 +457,15 @@ private void BrowseButton_Click(object sender, EventArgs e)
...
+        {
+
+        }
+
+        private void downloadingProgressLabel_Click(object sender, EventArgs e)
```
Comment (danil179) [inline League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs]: Delete this and its corresponding event. If you think that these are important please add `TODO:` that explains what is planned here.

**Review by danil179 — CHANGES_REQUESTED** [review]
Remove unused code.

**Review by danil179 — APPROVED** [review]
Approved.

**Pull request merged**